models/netsuite2/netsuite2__transaction_details.sql

```diff
@@ -50,10 +50,12 @@ vendors as (
     from {{ var('netsuite2_vendors') }}
 ),
 
+{% if var('netsuite2__using_vendor_categories', true) %}
```

Comment:
Hello there, may I ask why the **netsuite2__transaction_details** is not calling the variable for `netsuite2__multibook_accounting_enabled` too? In my case, with 2 books, all my transactions are duplicated but the **accounting_book_id** is not propagated.

Reply:
Hi @erwann-pleo thanks for commenting here. We actually are working on a more robust implementation for multi-book accounting to factor into the end models. You can see the progress in PR #90. Would you be able to attempt to use that branch version of the package and let me know if that resolves the duplicate errors you are seeing?

Reply:
You can use this branch by providing the following in your packages.yml in place of the official netsuite version:

```yml
packages:
  - git: https://github.com/jmongerlyra/dbt_netsuite.git
    revision: lyra_fixes
    warn-unpinned: false 
```